### PR TITLE
Update avo slug

### PR DESCRIPTION
**Pull request by bjornj12**
_A summary of your pull request, including the what change you're making and why._

Updating recently created **avo** destination, to have the slug **avo** instead of **actions-avo**,  so that the url is `.../avo`

Renaming the action to "Track schema from event"

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Comment by joe-ayoub-segment** [comment]
hi @bjornj12 - thanks for raising the PR. We're unable to change the Slug, but your other changes are ok!
The slug isn't something the customer will see in the UI anyway. 
I hope that's OK!

File: packages/destination-actions/src/destinations/avo/index.ts [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 import type { Settings } from './generated-types'
-import sendSchemaAction from './sendSchemaToInspector'
-import { Environment } from './sendSchemaToInspector/avo-types'
+import trackSchemaFromEvent from './trackSchemaFromEvent'
+import { Environment } from './trackSchemaFromEvent/avo-types'
 
 const destination: DestinationDefinition<Settings> = {
   name: 'Avo',
-  slug: 'actions-avo',
+  slug: 'avo',
```
Comment (logason) [inline packages/destination-actions/src/destinations/avo/index.ts:10]: If we want this to appear as "Catalog / Avo" in the UI, should `avo` here be capitalized, or is the `name` field used for that?

**Comment by bjornj12** [comment]
> hi @bjornj12 - thanks for raising the PR. We're unable to change the Slug, but your other changes are ok!
The slug isn't something the customer will see in the UI anyway.
I hope that's OK!

Hi @joe-ayoub-segment, is it not possible for us to have `catalog/avo` ?

Do i need to modify this PR in anyway before merge based on this?

**Comment by bjornj12** [comment]
@joe-ayoub-segment, I understand, we'd love to have catalog/avo, but ill revert the slug for now at least.

File: packages/destination-actions/src/destinations/avo/index.ts (hunk 1 above)
Comment (joe-ayoub-segment) [reply packages/destination-actions/src/destinations/avo/index.ts]: > If we want this to appear as "Catalog / Avo" in the UI, should `avo` here be capitalized, or is the `name` field used for that?

The slug is only used in the URL. It's the 'name' which is visible in the UI.

**Comment by joe-ayoub-segment** [comment]
Hi @bjornj12 - 

Looks like you want to update the name of the Action which is visible to the user in the Segment UI. And you also want to update the title of a field in that Action ( to indicate createdAt). 

I've created this separate PR which does this https://github.com/segmentio/action-destinations/pull/1866

I'd prefer to avoid creating a new Action as it would mean that I will need to run a DB script in Production to delete the old Action from our Control Plane DB. 

PR 1866 will update the UI titles for the Action and Field. 

Are you happy for me to progress with PR 1866 instead?

Best regards,
Joe

**Pull request closed without merging**